timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -162,15 +165,18 @@ void BDExchangeMove<RealType>::move(
     curandErrchk(templateCurandUniform(cr_rng_translations_, d_translations_.data, d_translations_.length));
     curandErrchk(templateCurandUniform(cr_rng_samples_, d_sample_noise_.data, d_sample_noise_.length));
     curandErrchk(templateCurandUniform(cr_rng_mh_, d_mh_noise_.data, d_mh_noise_.length));
-    for (int step = 0; step < steps_per_move_; step++) {
+    size_t step = 0;
```

Comment:
does `step` become redundant with `p_noise_offset_.data`?

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1274/commits/28b3dca9f1c602bd7493ac41902009fef13cde24